Celigo.ServiceManager.NetSuite.REST/src/Celigo.ServiceManager.NetSuite.REST.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-      <TargetFramework>net6.0</TargetFramework>
-      <Version>2.0.0</Version>
+      <TargetFramework>net8.0</TargetFramework>
```

Comment:
Was the upgrade mandatory? In the past we've encountered customers who needed the new WSDL but didn't have the option to upgrade their .net code base. So, I'd be cautious about bumping up the version.

Reply:
We had an internal discussion and we are moving this back to the earlier .net versions